src/lightning/pytorch/trainer/trainer.py

```diff
@@ -454,6 +454,10 @@ def __init__(
             GradClipAlgorithmType(gradient_clip_algorithm.lower()) if gradient_clip_algorithm is not None else None
         )
 
+        if detect_anomaly:
+            rank_zero_info(
+                "You have turned on detect_anomaly flag, which can significantly slow down compute speed and is recommended only for model debugging."
```

Comment:
```suggestion
                "You have turned on `Trainer(detect_anomaly=True)`. This will significantly slow down compute speed and is recommended only for model debugging."
```